R/utils.R

```diff
@@ -678,3 +678,52 @@ is_named = function(x) {
   }
   TRUE
 }
+
+# Used in parquet write/sink
+translate_statistics = function(statistics) {
+  if (is.logical(statistics)) {
```

Comment:
I think `length(statistics) == 1L` is needed for before checking the vector type.